tests/decimal_tests.rs

```diff
@@ -648,6 +648,275 @@ fn it_can_round_to_2dp() {
     assert_eq!("6.12", b.to_string());
 }
 
+#[test]
+fn it_can_round_to_2dp_using_bankers_rounding_strategy() {
+    let a = Decimal::from_str("6.12345").unwrap();
```

Comment:
On a side note, I've been more inclined to use test arrays going forward simply because it makes it easier to add tests. I think you could do probably set up something similar here to set precedent:
```
using RoundingStrategy::*;
let tests = &[
    ("6.12345", 2, BankersRounding, "6.12"),
    ("6.126", 2, BankersRounding, "6.13"),
    // etc
];
for &(input, dp, strategy, expected) in tests {
    let a = Decimal::from_str(input).unwrap();
    let b = a.round_dp_with_strategy(dp, strategy);
    assert_eq!(expected, b.to_string());
}
```
The odd one out of course is the test marked `// Issue #71`. For this one it could have a separate combined test which iterates the various rounding strategies on this irrational number.

Of course I realize that you're reusing an existing pattern - it's just one I'd like to move away from at some stage in the future. Whether that's now or later is up for debate 😄